src/content.js

```diff
@@ -609,13 +609,6 @@ function init() {
 		}
 	});
 
-	// Handle issue list ajax
```

Comment:
You've removed this comment when moving the function. Could we keep it?

Reply:
Restored a generic version of it, I'm planning on merging the two nearby `gitHubInjection` functions once we merge a few of the open PRs.